radio/src/lua/api_mavlink.cpp

```diff
@@ -0,0 +1,224 @@
+/*
+ *  (c) www.olliw.eu, OlliW, OlliW42
+ */
+
+#include <ctype.h>
+#include <stdio.h>
+#include "opentx.h"
+#include "lua_api.h"
+#include "thirdparty/Mavlink/edgetx_lua_lib_constants.h"
+#include "thirdparty/Mavlink/edgetx_lua_lib_messages.h"
+
+
+static int luaMavlinkGetVersion(lua_State * L)
+{
+  lua_pushinteger(L, FASTMAVLINK_MAVLINK_VERSION); // this is the version reported also by the heartbeat
+  return 1;
+}
+
+static int luaMavlinkGetChannelStatus(lua_State * L)
+{
+  lua_createtable(L, 0, 6);
+  lua_pushtableinteger(L, "msg_rx_count", mavlinkTelem.msg_rx_count);
+  lua_pushtableinteger(L, "msg_rx_per_sec", mavlinkTelem.msg_rx_persec);
+  lua_pushtableinteger(L, "bytes_rx_per_sec", mavlinkTelem.bytes_rx_persec);
+  lua_pushtableinteger(L, "msg_tx_count", mavlinkTelem.msg_tx_count);
+  lua_pushtableinteger(L, "msg_tx_per_sec", mavlinkTelem.msg_tx_persec);
+  lua_pushtableinteger(L, "bytes_tx_per_sec", mavlinkTelem.bytes_tx_persec);
+  return 1;
+}
+
+//-- some statistics --
```

Comment:
Functions in this section are no way related to MAVLink and should be part of another PR.